syntax = "proto2";
import "session_commands.proto";

message Command_ReplayModifyMatch {
    extend SessionCommand {
        optional Command_ReplayModifyMatch ext = 1102;
    }
    optional sint32 game_id = 1 [default = -1];
    optional bool do_not_hide = 2;
}
 
